src/git/util.js

```diff
@@ -45,7 +52,10 @@ export const createBranchName = R.curryN(
 export const getProject = R.compose(R.nth(1), R.match(/\/((\w|-)+)$/));
 
 export const getIssueIdFromBranch = R.compose(
```

Comment:
I think this function needs to be updated. The conversion of the branch template to create the regex looks good,
but this function was initially accepting one parameter (the branch name) and now it is taking two: the branch name and the config. 

I think something like this should work:

```JS
export const getIssueIdFromBranch = R.curryN(
  2,
  R.compose(
    R.converge(R.compose(R.ifElse(R.isNil, R.identity, R.toUpper), R.last, R.match), [
      R.compose( // This returns the regex
        R.reduce(
          (msg, [k, v]) => R.replace(`{${k}}`, v, msg),
          R.__, // Here goes the template
          R.toPairs(TEMPLATE_KEY_MATCHERS)
        ),
        getTemplate, // New function that I suggested above
        R.nthArg(0) // Config
      ),
      R.nthArg(1), // Branch name
    ])
  )
);
```